test/anki/hook_vendor_spec.py

```diff
@@ -8,13 +8,15 @@
 from crowd_anki.anki.hook_vendor import HookVendor
 from crowd_anki.config.config_settings import ConfigSettings
 
+
 def setup_vendor(automated_snapshot):
-    hook_manager = MagicMock()
+    h = MagicMock()
```

Comment:
why these renamings?

Reply:
PEP violation, they were "shadow names" of the function which called them. Open to a better name for those object, but it's just tests so I didn't care much.

Reply:
good names are important everywhere ;) 

how about `hook_vendor_mock` `hook_manager_mock`?